Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs

```diff
@@ -47,6 +47,10 @@ public enum ClearScreenOptions
     /// </summary>
     public const string CSI = "\u001B[";
 
+
+    public const string CSI_Device_Attributes_Request = CSI + "c";
+    public const string CSI_Device_Attributes_Request_Terminator = "c";
```

Comment:
There is already fields defined for these commands:

`CSI_SendDeviceAttributes` which is the correct command because if p1 is omitted then 0 is assumed.
`CSI_ReportDeviceAttributes_Terminator`

Reply:
Thanks, have reverted my additions here 👍